components/serializer.rst

```diff
@@ -628,6 +628,40 @@ processes::
     $anne = $normalizer->denormalize(['first_name' => 'Anne'], 'Person');
     // Person object with firstName: 'Anne'
 
+.. _using-underscored-method-names-for-camelized-attributes:
+
+snake_case to CamelCase
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Symfony provides a built-in name converter designed to transform between
+snake_case and CamelCased styles during serialization and deserialization
+processes::
+
+    use Symfony\Component\Serializer\NameConverter\SnakeCaseToCamelCaseNameConverter;
+    use Symfony\Component\Serializer\Normalizer\ObjectNormalizer;
+
+    $normalizer = new ObjectNormalizer(null, new SnakeCaseToCamelCaseNameConverter());
+
+    class Person
+    {
+        public function __construct(
+            private string $firstName,
+        ) {
+        }
+
+        public function getFirstName(): string
+        {
+            return $this->firstName;
+        }
+    }
+
+    $kevin = new Person('Kévin');
+    $normalizer->normalize($kevin);
+    // ['first_name' => 'Kévin'];
+
+    $anne = $normalizer->denormalize(['first_name' => 'Anne'], 'Person');
```

Comment:
The example here seems to be the same as the one for the `CamelCaseToSnakeCaseNameConverter`. That doesn't look right to me.